src/detect-flowbits.c

```diff
@@ -182,6 +182,20 @@ static int DetectFlowbitParse(char *str, char *cmd, int cmd_len, char *name,
             SCLogError(SC_ERR_PCRE_GET_SUBSTRING, "pcre_copy_substring failed");
             return 0;
         }
+
+        /* Trim trailing whitespace. */
+        while (isblank(name[strlen(name) - 1])) {
```

Comment:
the regex should prevent it, but it may be good to add a strlen(name) > 0 condition in the while loop. If strlen returns 0 we may have a negative index. Might prevent future issues if the regex changes.